### DISCUSSION TITLE
Add support for `realpath` cache

### DISCUSSION THREAD

**Discussion by kiler129** [shinsenter/php#20] [Ideas]
Hi!

Thank you for an amazing work! If you have a minute, it would be great to get two additional env vars for `realpath_cache_size` and `realpath_cache_ttl` settings.

This is per https://symfony.com/doc/current/performance.html recommendations, but these need to be increased in most production apps.

**Comment by shinsenter** [comment]
@kiler129 

The Docker images are built with a setting called [open_basedir](https://www.php.net/manual/ini.core.php#ini.open-basedir) enabled. This setting limits the areas of the file system that your PHP scripts are allowed to access. This helps to improve security by preventing scripts from accessing sensitive files outside of the specified directories. It also helps to limit disk usage by only allowing access to certain parts of the file system that your PHP scripts should be using.

Usually, when open_basedir is enabled, the realpath cache is disabled in PHP. The realpath cache is a way to speed up file path resolution by caching the resolved path of files and directories. But, in this case, because the Docker images have the OpCache extension enabled in PHP, so you don't need to enable `realpath_cache_size` and `realpath_cache_ttl` as these settings for `realpath_cache` will have no effect.
